benchmarks/src/main/java/io/druid/benchmark/datagen/BenchmarkSchemas.java

```diff
@@ -92,5 +92,6 @@
         basicSchemaDataInterval
     );
     SCHEMA_MAP.put("basic", basicSchema);
+    SCHEMA_MAP.put("nested", basicSchema);
```

Comment:
you can remove this after moving "nested" to be a query under the "basic" schema